cli/medperf/comms/entity_resources/sources/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .direct import DirectLinkSource
+from .synapse import SynapseSource
+
+supported_sources = [DirectLinkSource, SynapseSource]
+
+__all__ = [supported_sources]
```

Comment:
[Check this](https://stackoverflow.com/questions/44834/what-does-all-mean-in-python)
I changed how our `__all__`'s are defined. It seems we don't actually get the benefit of it since we don't use star imports, but it is useful to suppress flake's errors of `imported but unused`.